### ISSUE TITLE
Either stream expression doesn't match, or it does match and produces nothing

### ISSUE THREAD

**Issue by CMCDragonkai** [jorgebastida/awslogs#223]
I have a log group as `/ecs/inverter-analysis`, and log stream as `ecs/inverter-analysis-gpu/6258f788-9849-4836-9332-7e7dd96d6702`.

When I run:

```
awslogs get '/ecs/inverter-analysis' 'ecs/inverter-analysis-gpu/6258f788-9849-4836-9332-7e7dd96d6702'
```

It gives me back:

```
No streams match your pattern 'ecs/inverter-analysis-gpu/6258f788-9849-4836-9332-7e7dd96d6702'.
```

So then I try something like:

```
awslogs get '/ecs/inverter-analysis' '.*7e7dd96d6702' --aws-region=ap-southeast-2 --profile=formbay
```

And that also just says `No streams match your pattern '.*7e7dd96d6702'.`.

Then I tried just `.*`, and it produced nothing, no error.

Then after I waited 30 minutes, and reran the same command, the command started producing output and didn't complain of any error. The log already existed and had millions of entries.

**Comment by danieltorrecillas** [comment]
I've experience this too, with the same "resolution"—wait a little while and the same command suddenly works.

**Comment by cykl** [comment]
See issues: #96, #143, #195 
Pending fixes: #199, #185 (@blueyed fix seems (simpl|bett)er than mine for most use cases)

**Comment by CMCDragonkai** [comment]
Is this resolved by #199?

**Comment by jorgebastida** [comment]
Yes, ``--watch`` is not working well. I'm making this as dup, I'll be using https://github.com/jorgebastida/awslogs/issues/235 as the reference ticket to solve this issue.
